cmd/workspace_test.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/astronomer/astro-cli/config"
+	"github.com/astronomer/astro-cli/houston"
+	"github.com/astronomer/astro-cli/pkg/httputil"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+// RoundTripFunc
+type RoundTripFunc func(req *http.Request) *http.Response
+
+// RoundTrip
+func (f RoundTripFunc) RoundTrip(req *http.Request) (*http.Response, error) {
+	return f(req), nil
+}
+
+// NewTestClient returns *httputil.HTTPClient with Transport replaced to avoid making real calls
+func NewTestClient(fn RoundTripFunc) *httputil.HTTPClient {
+	testClient := httputil.NewHTTPClient()
+	testClient.HTTPClient.Transport = RoundTripFunc(fn)
+	return testClient
+}
+
+func TestWorkspaceList(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	configYaml, err := ioutil.ReadFile("./testdata/config.yaml")
+	err = afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
+	config.InitConfig(fs)
+	expectedOut := " NAME           ID                            \n\x1b[1;32m airflow        ck05r3bor07h40d02y2hw4n4v     \x1b[0m\n airflow123     XXXXXXXXXXXXXXX               \n"
```

Comment:
COuld you format this over multiple lines too so it's a bit easier to read?

```suggestion
	expectedOut := ` NAME           ID                            
\x1b[1;32m airflow        ck05r3bor07h40d02y2hw4n4v     \x1b[0m
 airflow123     XXXXXXXXXXXXXXX               
 `
```

(I forget if there's a way in Go stdlib to "de-dent" things like python's textwrap.dedent)

Reply:
best i can do:
```
	expectedOut := " NAME           ID                            \n" +
		"\x1b[1;32m airflow        ck05r3bor07h40d02y2hw4n4v     \x1b[0m\n " +
		"airflow123     XXXXXXXXXXXXXXX               \n"
```